task2.js

```diff
@@ -0,0 +1,27 @@
+paragraph = "Developing writers can often benefit from examining an essay, a paragraph, or even a sentence to determine what makes it effective."
+
+function getFrequency(string) {
```

Comment:
`getFrequency` is not conveying complete sense.

Reply:
computeFrequency now